### PR TITLE
Added jax.numpy.csingle

### PR DESCRIPTION
**Pull request by Dev-Khant**
Solves #14413

### CODE REVIEW

**Comment by Dev-Khant** [comment]
Hello @hello-fri-end, locally all the tests are passed but I don't why its failing here. So if something is wrong do let me know.
Thanks

File: ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_creation.py [hunk 1]
```diff
@@ -812,6 +812,28 @@ def test_geomspace(
     )
 
 
+@handle_frontend_test(
+    fn_tree="jax.numpy.csingle",
+    dtype_and_x=helpers.dtype_and_values(available_dtypes=helpers.get_dtypes("complex"))
```
Comment (hello-fri-end) [inline ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_creation.py:817]: looks like `jax.numpy.csingle` is an alias of `jax.numpy.complex64`. You should add `aliases=["jax.numpy.complex64"]` in the decorator :)

**Review by hello-fri-end — CHANGES_REQUESTED** [review]
Thanks for making this PR @Dev-Khant :rocket:. Just left a few minor comments - once you resolve those, we can merge this :)

**Review by hello-fri-end — CHANGES_REQUESTED** [review]
Can you please look at the formatting errors? @Dev-Khant

**Comment by Dev-Khant** [comment]
@hello-fri-end I have made the changes and those lints errors are from other files.

**Review by hello-fri-end — APPROVED** [review]

**Comment by Dev-Khant** [comment]
@hello-fri-end If everything is seems good, can you merge it?

**Comment by hello-fri-end** [comment]
> @hello-fri-end If everything is seems good, can you merge it?
Was waiting for the jobs to finish. Thanks for the contribution. :)

**Comment by Dev-Khant** [comment]
Thanks for Merging :)

**Pull request merged**